### PR TITLE
add btn-group size variants

### PR DESCRIPTION
**Pull request by JefMari**
PR for issue https://github.com/dev-onenetworkecommerce/aia/issues/35
Size Reference is the same as Size Buttons

You may also find that i add `margin-left: -1px` in to each adjacent sibling to create a clean `btn-group` look

Before:

After adding `margin-left: -1px` :

### CODE REVIEW

File: styles/components/button.scss [hunk 1]
```diff
@@ -133,16 +133,31 @@
...
+    height: 30px;
+    font-size: $mini;
+    line-height: 30px;
+  }
+  &.-large > .btn {
+    height: 50px;
+    padding: 0 25px;
+    font-size: $tiny;
+    line-height: 40px;
+  }
```
Comment (srph) [inline styles/components/button.scss:153]: IMO, why not:

``` sass
&.-large > .btn {
  @extend .btn.-large;
}
```

This way, adjusting is easy peezy. Plus it's DRY :star:

\* The same for `-small` :100:

**Comment by srph** [comment]
> You may also find that i add `margin-left: -1px` in to each adjacent sibling to create a clean `btn-group` look

Good job taking note on the detail here :+1:

File: styles/components/button.scss (hunk 1 above)
Comment (JefMari) [inline styles/components/button.scss:153]: oh forgot about being DRY. haha

**Comment by JefMari** [comment]
Done with the reviews :+1:

**Comment by srph** [comment]
How about for extra small?

**Comment by JefMari** [comment]
Heads up!
Here's an update!

**Comment by srph** [comment]
THAAANKS

**Pull request merged**